src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,21 @@
+package wolox.training.repositories;
+
+import java.util.Optional;
+import org.springframework.data.repository.Repository;
+import wolox.training.models.Book;
+
+/**
+ * A repository for {@link Book}s.
+ */
+@org.springframework.stereotype.Repository
```

Comment:
Because there is another class called Repository (i.e the extended interface) so, as the annotation has the same name, then I need to use the fully qualified name of the annotation class.